### DISCUSSION TITLE
Additional confessions

### DISCUSSION THREAD

**Discussion by NonlinearFruit** [pmachapman/Confessions#6] [General]
I love the app and I've been a long time user. If you are interested I've collected a variety of Reformed creeds/confessions/catechisms here:
https://github.com/NonlinearFruit/Creeds.json

It also has a list of related resources on this topic: apps (like your android app), data sources, books, websites.

I'd love to here any recommendations/feedback you have. I've been over the moon with the recent app updates you've made! Keep up the good work.

**Comment by pmachapman** [comment]
Thanks for getting in touch! In the coming week or so I will publish my translation of the Barmen Declaration here - it is CC BY 4.0, so you will have no issue using it in your repo.

You may find the [Confessions-Indexer](https://github.com/pmachapman/Confessions-Indexer/) project useful. This takes the data from this repo and outputs an MSSQL or SQLite database. That should be easy for you to then convert to JSON.

I am also working on an iOS port of the app, and there are always requests for more Confessions and Catechisms, which I will get to eventually - your data will be really useful for that!
